cromwell.examples.conf

```diff
@@ -212,6 +212,17 @@ backend {
         # the job and then immediately return a scheduled job id.
         run-in-background = true
 
+        # `temporary-directory` creates the temporary directory for commands.
+        #
+        # If this value is not set explicitly, the default value creates a unique temporary directory, equivalent to:
+        # temporary-directory = "$(mktemp -d \"$PWD\"/tmp.XXXXXX)"
+        #
+        # The expression is run from the execution directory for the script. The expression must create the directory
+        # if it does not exist, and then return the full path to the directory.
+        #
+        # To create and return a constant temporary directory, use something like:
+        # temporary-directory = "$(mkdir -p /tmp/mydir && echo /tmp/mydir)"
```

Comment:
Hopefully users wouldn't expect this to be visible across Docker invocations (at least without poking in new volume mounts)?

Reply:
And perhaps a counterindication for Mac whose Docker I believe doesn't like volumes mounted in that are not under the home directory?

Reply:
Oops, I didn't mean constant as in reusable, but constant as in a non-random path. The example above was meant to ensure that config editors realize when not using the default `mktemp` that they must still create the directory iff it doesn't exist, and return the path as a string. I'll edit the sentence.

Most folks should not need to edit this config value. See the original ticket #2655 for the use cases.